src/tyr/route_serializer_osrm.cc

```diff
@@ -96,9 +98,11 @@ const constexpr PointLL::first_type DOUGLAS_PEUCKER_THRESHOLDS[19] = {
     2.6,      // z18
 };
 
-const constexpr double SECONDS_BEFORE_VERBAL_TRANSITION_ALERT_INSTRUCTION = 15.0;
-const constexpr double SECONDS_BEFORE_VERBAL_PRE_TRANSITION_INSTRUCTION = 5.0;
+const constexpr double SECONDS_BEFORE_VERBAL_TRANSITION_ALERT_INSTRUCTION = 35.0;
+const constexpr double SECONDS_BEFORE_VERBAL_PRE_TRANSITION_INSTRUCTION = 10.0;
+const constexpr double MIN_DISTANCE_VERBAL_PRE_TRANSITION_INSTRUCTION = 40;
 const constexpr double APPROXIMATE_VERBAL_POSTRANSITION_LENGTH = 110;
+const constexpr double APPROXIMATE_VERBAL_PRERANSITION_SECONDS = 3.0;
```

Comment:
Not a blocker for this PR since it was already hard-coded, but I *think* it would be pretty easy to make these configurable (with defaults) at request time.

Reply:
Yes, that shouldn't be a problem. This would make it more flexible for everybody to adapt it to their needs. We should have a follow up PR for this